### ISSUE TITLE
If journal.txt file does not exist, journal.rb crashes on new action

### ISSUE THREAD

**Issue by greenmanspirit** [greenmanspirit/journal.txt#1]
run `journal` without journal.txt file existing and journal.rb fails after you save and exit the editor. 

Ruby error

```
journal.txt/journal.rb:40:in `initialize': No such file or directory - journal.txt (Errno::ENOENT)
    from /home/adam/journal.txt/journal.rb:40:in `new'
    from /home/adam/journal.txt/journal.rb:40:in `new_entry'
    from /home/adam/journal.txt/journal.rb:160:in `<main>'
```

**Comment by greenmanspirit** [comment]
Solved by making the journal object initialize method check to see if the file exists. If it does not, it asks the user if they want to create it and creates the file with a blank like if yes. If they say no, says it cannot work without a file and exits. I set the default to no so that any keystroke other than y or Y does not create the file to try and accommodate for mistakes like bad config without accidentally creating a new file.
